utils.py

```diff
@@ -115,7 +161,7 @@ def generate_data_dashboard_link(sz):
     color = sz["color"].lower()
     stop1 = sz["fr_id"]
     stop2 = sz["to_id"]
-    start = (datetime.strptime(sz["start"], "%Y-%m-%dT%H:%M:%S") - timedelta(days=14)).strftime("%Y-%m-%d")
+    start = (datetime.strptime(sz["start"], "%Y-%m-%dT%H:%M:%SZ") - timedelta(days=14)).strftime("%Y-%m-%d")
```

Comment:
Same comment here as @devinmatte's above

Reply:
I think I was on the wrong python version 👍 removed the Z and everything is fine